simde/simde-arch.h

```diff
@@ -178,6 +178,47 @@
 #  define SIMDE_ARCH_H8300
 #endif
 
+/* Elbrus (8S, 8SV and successors)
+   <https://en.wikipedia.org/wiki/Elbrus-8S> */
+#if defined(__e2k__)
+#define SIMDE_ARCH_E2K
+#define SIMDE_SKIP_EXTENDED_E2K_VECTOR_OPS /* Discard features unsupported by compiler */
+#endif
+
+/* SIMD ISA extensions for Elbrus (equivalent to X86 ones) */
+#if defined(SIMDE_ARCH_E2K)
+#if defined(__MMX__)
+#define SIMDE_ARCH_X86_MMX 1
+#endif
+#if defined(__SSE__)
+#define SIMDE_ARCH_X86_SSE 1
+#endif
+#if defined(__SSE2__)
+#define SIMDE_ARCH_X86_SSE2 1
+#endif
+#if defined(__SSE3__)
+#define SIMDE_ARCH_X86_SSE3 1
+#endif
+#if defined(__SSSE3__)
+#define SIMDE_ARCH_X86_SSSE3 1
+#endif
+#if defined(__SSE4_1__)
+#define SIMDE_ARCH_X86_SSE4_1 1
+#endif
+#if defined(__SSE4_2__)
+#define SIMDE_ARCH_X86_SSE4_2 1
+#endif
+#if defined(__AVX__)
+#define SIMDE_ARCH_X86_AVX 1
+#endif
+#if defined(__AVX2__)
+#define SIMDE_ARCH_X86_AVX2 1
+#endif
+#if defined(__PCLMUL__)
+#define SIMDE_ARCH_X86_PCLMUL 1
+#endif
+#endif
+
```

Comment:
Yes, except for some uncertain cases, like that LCC incorrectly defines `__FMA__` and `__XOP__` while actually has incomplete support for it. Fixed in too. Probably if it will be fixed in some newer version of compiler, here may come a condition based on `__LCC__` value.